tests/acceptance/SettingsUnsubscribePageCest.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+class SettingsUnsubscribePageCest {
+  function previvewDefaultUnsubscribePage(\AcceptanceTester $I) {
+    $I->wantTo('Preview default MailPoet Unsubscribe page from MP Settings page');
+    $I->login();
+    $I->amOnMailPoetPage('Settings');
+    $I->click(['xpath'=>'//*[@id="mailpoet_settings_form"]/div[2]/table/tbody/tr[6]/td/p/a']);
```

Comment:
I think it would be nicer and more robust to use a data-automation attribute. For this case you can add it quite simply here:
https://github.com/mailpoet/mailpoet/blob/b21ef30202c1ab3f88682340c1b8662d37670c9b/views/settings/basics.html#L294